docs/administration/setting-job-flags.md

```diff
@@ -30,3 +32,28 @@ Unset:
 ```bash
 irm /tempZone/yoda/flags/stop_revisions
 ```
+
+## For version > 1.8
+
+### Replication flag
+
+Set:
+```bash
+imkdir /tempZone/yoda/flags/stop_replication
+```
+
+Unset:
+```bash
+irm /tempZone/yoda/flags/stop_replication
```

Comment:
I think this irm command needs a -r option, because the parameter is a collection